torchtune/modules/transformer.py

```diff
@@ -158,6 +158,10 @@ def setup_caches(self, max_batch_size: int, dtype: torch.dtype) -> None:
             torch.ones(self.max_seq_len, self.max_seq_len, dtype=torch.bool)
         )
 
+    def reset_caches(self):
```

Comment:
Does it need a docstring?

Reply:
Just following protocol with setup_caches. I think it's pretty self explanatory, but I don't mind adding one if people want.